### ISSUE TITLE
Problem: Docker build broken by #290

### ISSUE THREAD

**Issue by weex** [magicstone-dev/ecko#306]
### Please describe the problem

After merging #290 see the following build error on docker hub.

```
#13 1.455 The path `/opt/mastodon/gems/ecko-plugins` does not exist.
#13 ERROR: executor failed running [/bin/bash -c cd /opt/mastodon && bundle config set deployment 'true' && bundle config set without 'development test' && bundle config set silence_root_warning true && bundle install -j"$(nproc)" && yarn install --pure-lockfile]: exit code: 13
------
> [build-dep 7/9] RUN cd /opt/mastodon && bundle config set deployment 'true' && bundle config set without 'development test' && bundle config set silence_root_warning true && bundle install -j"$(nproc)" && yarn install --pure-lockfile:
------
error: failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/bash -c cd /opt/mastodon && bundle config set deployment 'true' && bundle config set without 'development test' && bundle config set silence_root_warning true && bundle install -j"$(nproc)" && yarn install --pure-lockfile]: exit code: 13
Build failed using Buildkit
```

https://hub.docker.com/repository/registry-1.docker.io/dsterry/ecko/builds/c636b9cf-2ac2-46b1-a2ab-cdc5f15656dc

**Comment by weex** [comment]
Failed for me locally on a `docker build .`

Tried this change to Dockerfile

```
-COPY Gemfile* package.json yarn.lock /opt/mastodon/
+COPY Gemfile* package.json yarn.lock gems/ecko-plugins/ /opt/mastodon/
```

But I then get...
```
$ docker build --no-cache .
[+] Building 184.0s (12/18)                                                                                                                             
 => [internal] load build definition from Dockerfile                                                                                               0.0s
 => => transferring dockerfile: 37B                                                                                                                0.0s
 => [internal] load .dockerignore                                                                                                                  0.0s
 => => transferring context: 35B                                                                                                                   0.0s
 => [internal] load metadata for docker.io/library/ubuntu:20.04                                                                                    0.5s
 => [internal] load build context                                                                                                                  0.6s
 => => transferring context: 980.26kB                                                                                                              0.6s
 => CACHED [stage-1 1/6] FROM docker.io/library/ubuntu:20.04@sha256:626ffe58f6e7566e00254b638eb7e0f3b11d4da9675088f4781a50ae288f3322               0.0s
 => [stage-1 2/6] RUN echo "Etc/UTC" > /etc/localtime &&  apt-get update &&  apt-get install -y --no-install-recommends whois wget    libssl1.1  135.8s
 => [build-dep 2/9] RUN echo "Etc/UTC" > /etc/localtime &&  apt-get update &&  apt-get install -y --no-install-recommends  bison build-essential  58.2s
 => [build-dep 3/9] RUN ARCH= &&     dpkgArch="$(dpkg --print-architecture)" &&   case "${dpkgArch##*-}" in     amd64) ARCH='x64';;     ppc64el)  12.4s
 => [build-dep 4/9] RUN cd ~ &&  wget https://cache.ruby-lang.org/pub/ruby/${RUBY_VER%.*}/ruby-$RUBY_VER.tar.gz &&  tar xf ruby-$RUBY_VER.tar.g  106.5s
 => [build-dep 5/9] RUN npm install -g yarn &&  gem install bundler                                                                                5.1s
 => [build-dep 6/9] COPY Gemfile* package.json yarn.lock gems/ecko-plugins/ /opt/mastodon/                                                         0.2s 
 => ERROR [build-dep 7/9] RUN cd /opt/mastodon &&   bundle config set deployment 'true' &&   bundle config set without 'development test' &&   bu  0.9s 
------                                                                                                                                                  
 > [build-dep 7/9] RUN cd /opt/mastodon &&   bundle config set deployment 'true' &&   bundle config set without 'development test' &&   bundle config set silence_root_warning true &&  bundle install -j"$(nproc)" &&  yarn install --pure-lockfile:                                                           
#12 0.872 fatal: not a git repository (or any of the parent directories): .git                                                                          
#12 0.903 Your bundle only supports platforms ["x86_64-darwin-20"] but your local platform                                                              
#12 0.903 is x86_64-linux. Add the current platform to the lockfile with `bundle lock                                                                   
#12 0.903 --add-platform x86_64-linux` and try again.                                                                                                   
------                                                                                                                                                  
executor failed running [/bin/bash -c cd /opt/mastodon &&   bundle config set deployment 'true' &&   bundle config set without 'development test' &&   bundle config set silence_root_warning true && 	bundle install -j"$(nproc)" && 	yarn install --pure-lockfile]: exit code: 16
```

**Comment by weex** [comment]
Fixed by #313
